### DISCUSSION TITLE
Change default configuration

### DISCUSSION THREAD

**Discussion by ghost** [benbusby/whoogle-search#199] [Q&A] [answered]
Hi, I was wondering where i could find default configuration settings in code so i could change them a bit. 
Like change dark mode to on on default, change root url etc,
Thank you

**Comment by benbusby** [comment, marked as answer]
Defaults are found here:

https://github.com/benbusby/whoogle-search/blob/6c85468cd62bca852554eeed6090c4804e873249/app/models/config.py#L3-L14

**Reply by ghost** [reply]
Thank you very much!
